sxg_rs/src/static/prefetch.html

```diff
@@ -0,0 +1,28 @@
+<!--
+Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<meta charset="utf-8">
+<p>Loading SXG test page in 2 seconds</p>
+<script>
+  const sxgUrl = `./success.sxg?q=${Math.random()}`;
+  const link = document.createElement('link');
```

Comment:
Why create the link in JS rather than in pure HTML?

Reply:
Because the URL contains a random number.

Reply:
Oh, right. :facepalm: